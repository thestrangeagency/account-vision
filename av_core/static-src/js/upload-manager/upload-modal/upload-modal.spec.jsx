import React from 'react';
import { shallow } from 'enzyme';

import UploadModal from './upload-modal';

test('it renders', () => {
  const wrapper = shallow(<UploadModal
    isSaveDisabled={false}
    onHidden={() => {}}
    onShown={() => {}}
    onSubmit={() => {}}
    setDescriptionRef={() => {}}
  />);
  expect(wrapper).toMatchSnapshot();
});
