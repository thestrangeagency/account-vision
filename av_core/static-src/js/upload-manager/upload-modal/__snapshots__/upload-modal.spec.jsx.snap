// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  aria-hidden="true"
  className="modal fade"
  role="dialog"
  tabIndex="-1"
>
  <div
    className="modal-dialog"
  >
    <div
      className="modal-content"
    >
      <form
        className="m-0"
        noValidate={true}
        onSubmit={[Function]}
      >
        <div
          className="modal-body"
        >
          <div
            className="form-group"
          >
            <label
              htmlFor="input_description"
            >
              Description
            </label>
            <textarea
              className="form-control"
              id="input_description"
              name="notes"
            />
          </div>
        </div>
        <div
          className="modal-footer"
        >
          <button
            className="btn btn-secondary"
            data-dismiss="modal"
            disabled={false}
            type="button"
          >
            Cancel
          </button>
          <button
            className="btn btn-primary"
            disabled={false}
            type="submit"
          >
            Update
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
